libtsuba/src/RDGCore.h

```diff
@@ -206,7 +202,8 @@ class KATANA_EXPORT RDGCore {
 
   katana::Result<void> RegisterTopologyFile(const std::string& new_top) {
     part_header_.set_topology_path(new_top);
-    return topology_file_storage_.Unbind();
+    return topology_file_storage_.Bind(
+        rdg_dir_.Join(new_top).string(), /*resolve =*/false);
```

Comment:
Just curious what is going on here.  This function went from calling `Unbind` to calling `Bind`.  Seems weird.